pkg/testutils/testutils.go

```diff
@@ -169,7 +169,7 @@ func EnsureServiceHealthy(t testing.TB, grpcAddr, httpAddr string, transportCred
 		grpc.WithConnectParams(grpc.ConnectParams{Backoff: grpcbackoff.DefaultConfig}),
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
Checking code I can't find the connection either yet.
Running locally I was not able to reproduce the error. 

If possible confirm if you could reproduce it on your local machine & how

Thank you

Reply:
I've never been able to reproduce locally, it's always in github actions unfortunately.

Reply:
I've run more tests pushing and checking GitHub and can't reproduce the issue anymore.

As my hypothesis was wrong, this change should not be merged. I'll move PR to draft in case I find more time to investigate & try to replicate